server/routes/serviceProviderReferrals/dashboardPresenter.ts

```diff
@@ -40,8 +48,16 @@ export default class DashboardPresenter {
 
   readonly navItemsPresenter = new PrimaryNavBarPresenter('Referrals', this.loggedInUser)
 
-  readonly tableRows: SortableTableRow[] = this.referralsSummary.map(referralSummary => {
+  readonly tableRows: SortableTableRow[] = this.sentReferrals.content.map(referralSummary => {
     const sentAtDay = CalendarDay.britishDayForDate(new Date(referralSummary.sentAt))
+    const interventionForReferral = this.interventions.find(
+      intervention => intervention.id === referralSummary.referral.interventionId
+    )
+    if (interventionForReferral === undefined) {
+      throw new Error(
+        `Expected referral to be linked to an intervention with ID ${referralSummary.referral.interventionId}`
```

Comment:
I think if this occurs, we would benefit from the `referral.id` as well, could we include that, please?